metrics/lib/features/dashboard/domain/repositories/metrics_repository.dart

```diff
@@ -3,12 +3,13 @@ import 'package:metrics/features/dashboard/domain/entities/core/project.dart';
 
 /// Base class for metrics repositories.
 ///
-/// Provides an ability to get the data.
+/// Provides an ability to get the metrics data.
 abstract class MetricsRepository {
   /// Provides the stream of [Project]s.
   Stream<List<Project>> projectsStream();
 
-  /// Provides the stream of [Build]s of the project with [projectId] limited by [limit] elements.
+  /// Provides the stream of [Build]s of the project with [projectId]
+  /// where only [limit] elements returned.
```

Comment:
Should we also state that it's ordered and only most recent [limit] elements returned?